sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - `BlobLeaseClient::Change()` updates internal lease id.
 - Removed `ContentType` from `GetBlockListResult`.
 - Moved `GetPageRangesResult` to detail namespace.
+- `BlobServiceClient::UndeleteBlobContainer` doesn't support restoring a deleted container under a different name anymore.
```

Comment:
This is driven primarily by the service-side behavioral change than the SDK, correct? Might be worth mentioning that.
```suggestion
- `BlobServiceClient::UndeleteBlobContainer` doesn't support restoring a deleted container under a different name because the Azure Blob Service doesn't support it.
```